serde/src/private/ser.rs

```diff
@@ -54,6 +54,7 @@ enum Unsupported {
     Sequence,
     Tuple,
     TupleStruct,
+    #[cfg(not(any(feature = "std", feature = "alloc")))]
```

Comment:
Why does this only exist without alloc?

Reply:
This variant used only when allocations are not available, rustc emits a warning about unused variant.

Reply:
Oh 🤦 I forgot to check the impl, thx